docs/src/main/tut/xor.md

```diff
@@ -7,3 +7,338 @@ scaladoc: "#cats.data.Xor"
 ---
 # Xor
 
+In day to day programming, it is fairly common to find ourselves writing functions that
+can fail. For instance, querying a service may result in a connection issue, or some
+unexpected JSON response.
+
+To communicate these errors it has become common practice to throw exceptions. However,
+exceptions are not tracked in any way, shape, or form by the Scala compiler. To see
+if a function throws an exception, and moreover what kind of exception it throws, we
+have to dig through the source code. Then to handle these exceptions, we have to make sure
+we catch them at the call site. This all becomes even more unwieldy when we try to compose
+exception-throwing procedures.
+
+```scala
+val throwsSomeStuff: Int => Double = ???
+
+val throwsOtherThings: Double => String = ???
+
+val moreThrowing: String => List[Char] = ???
+
+val magic = throwsSomeStuff.andThen(throwsOtherThings).andThen(moreThrowing)
+```
+
+Assume we happily throw exceptions in our code. Looking at the types, any of those functions
+can throw any number of exceptions, we don't know. When we compose, exceptions from any of
+the constituent functions can be thrown. Moreover, they may throw the same kind of exception
+(e.g. `IllegalArgumentException`) and thus it gets tricky tracking exactly where that
+exception came from.
+
+How then do we communicate an error? By making it explicit in the data type we return.
+
+## Xor
+
+### Why not `Either`
+`Xor` is very similar to `scala.util.Either` - in fact, they are isomorphic.
+
+```scala
+sealed abstract class Xor[+A, +B]
+
+object Xor {
+  final case class Left[+A](a: A) extends Xor[A, Nothing]
+  final case class Right[+B](b: B) extends Xor[Nothing, B]
+}
+```
+
+Just like `Either`, it has two type parameters. Instances of `Xor` either hold a value
+of one type parameter, or the other. Why then does it exist at all?
+
+Taking a look at `Either`, we notice immediately there is no `flatMap` or `map` method on
+it. Instead, we must call `Either#right` (or `Either#left`) and then call `flatMap` or `map`
+on the result of that. The result of calling `Either#right` is a `RightProjection`, the name
+indicating what direction the `Either` is "biased" in. `RightProjection` is "right-biased", so
+calling `flatMap` or `map` on a `RightProjection` acts on the right type-parameter.
+
+```tut
+val e1: Either[String, Int] = Right(5)
+e1.right.map(_ + 1)
+
+val e2: Either[String, Int] = Left("hello")
+e2.right.map(_ + 1)
+```
+
+Note the return types are themselves back to `Either`, so if we want to make more calls to
+`flatMap` or `map` then we again must call `right` or `left`.
+
+More often than not we want to just bias towards one side and call it a day - by convention,
+the right side is most often chosen. This is the primary difference between `Xor` and `Either` -
+it is right-biased by default. `Xor` also has some more convenient methods on it, but the most
```

Comment:
Maybe instead say "`Xor` is right-biased". I think the referent of "it" is possibly confusing, and I would leave out the "by default" since it might also be a bit misleading (it might imply that bias is a matter of configuration).